doc/extensions.rst

```diff
@@ -57,6 +62,75 @@ Once the new extension class has been defined, the openml-python module to
 :meth:`openml.extensions.register_extension` must be called to allow OpenML-Python to
 interface the new extension.
 
+The following methods should get implemented. Although the documentation in
+the `Extension` interface should always be leading, here we list some additional
+information and best practices. 
+The `Sklearn Extension API :class:`openml.extensions.sklearn.SklearnExtension.html`
+is a good example to follow. Note that most of the methods to be implemented 
+are relatively simple, and can be done in several lines of code. 
+
+* General setup (required)
+  * :meth:`can_handle_flow`: Takes as argument an OpenML flow, and checks
+    whether this can be handled by the current extension. The OpenML database
```

Comment:
The docbuild fails because this indentation will lead to a warning (which we don't allow). In case you have issues building docs locally there's also a [docker image](https://github.com/openml/openml-python/tree/develop/docker) available.

Reply:
should I remove all line breaks?

Reply:
Line breaks are OK, but it's indented too far for this parser. Content for the lists should only be indented as far as the asterisks (not sure why the parser wants this, to be honest).

Reply:
I think this is a misleading error message. I added a line break and it now works. I created a commit which fixes doc building and will now create a commit fixing the merge conflicts arising from this.